plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -565,6 +567,47 @@ private function set_search_args(): void {
 		}
 	}
 
+	/**
+	 * Implements filtering of orders by created_via value.
+	 */
+	private function set_created_via_args(): void {
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
+		$created_via = sanitize_text_field( wp_unslash( $_GET['_created_via'] ?? '' ) );
+
+		if ( empty( $created_via ) ) {
+			return;
+		}
+
+		$this->order_query_args['created_via'] = $created_via;
+		$this->has_filter                      = true;
+	}
+
+	/**
+	 * Render the created_via filter dropdown.
+	 *
+	 * @return void
+	 */
+	public function created_via_filter() {
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
+		$current_created_via = isset( $_GET['_created_via'] ) ? sanitize_text_field( wp_unslash( $_GET['_created_via'] ) ) : '';
+
+		$created_via_options = array(
```

Comment:
@malinajirka @joshuatf Should we have those two options for now? Should we make the `created_via_options` filterable? Any suggestions? 🤔 TY

Reply:
I think the two options might be enough for now. However, this makes me wonder whether `Site checkout` label isn't a bit misleading since it'll display even orders created via wp-admin 🤔 . Wdyt?

Reply:
I think orders manually created in wp-admin have `created_via`=`admin`, see my other [comment](https://github.com/woocommerce/woocommerce/pull/57273/files#r2046542171) 🤔